SparkPost/matchbox
# UX-428 Update cypress and testing library to latest

### What Changed
Cypress Changelog: https://docs.cypress.io/guides/references/changelog.html#6-0-0

Two main breaking changes that apply to us:
>Cypress now always throws an error when asserting on an element that doesn’t exist in the DOM (unless you’re asserting that the element should not.exist). Assertions such as not.visible, not.contains, not.have.class, etc will now fail when used on non-existent DOM elements. Addresses #205.

> DOM elements where the CSS style (or ancestors) are opacity: 0 are no longer considered visible. However these are still considered actionable and any action commands used to interact with the element will perform the action. This matches browser’s implementation on how they regard elements with opacity: 0. Addresses #4474.



### How To Test or Verify
- Verify integration tests pass
- Verify we are making correct use of `be.visible` and `exist`


### PR Checklist
